File: app/models/Victim.py
class Victim:
    id: int
    xCoordinate: float
    yCoordinate: float
    truePositive: bool = False
    locationChecked: bool = False

    def __init__(self, id):
        self.id = id
    
    def __init__(self, xCoordinate: float, yCoordinate: float):
        self.xCoordinate = xCoordinate
        self.yCoordinate = yCoordinate

File: rubble-rescue-ui/src/components/mapItems/Victim.js
import React from "react";
import classes from './Victim.module.css';

const Victim = (props) => {
    // let coords = props.victim['coordinates'].split(", ");
    let left = `${Math.floor(parseFloat(props.victim.xCoordinate) * 10) + 52.1}%`;
    let top = `${Math.floor(parseFloat(props.victim.yCoordinate) * 10) + 37.5}%`;
    
    const style = {"left": left, "top": top}

    return ( <img src={require('../../assets/victim.png')} className={classes.Image} style={style} alt="Victim" /> );
}

export default Victim;
